web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -115478,6 +115485,9 @@
     function Nr() {
         return interfaceConfig.RECENT_LIST_ENABLED
     }
+    function InProgressListEnabled() { // BAO
+        return interfaceConfig.IN_PROGRESS_LIST_ENABLED
+    }
```

Comment:
There is no need to make it a function, but I made it match RecentList ( Nr() ).